frontend/src/metabase/query_builder/components/view/ViewHeader.jsx

```diff
@@ -416,7 +421,12 @@ function ViewTitleHeaderRightSide(props) {
     >
       {hasSaveButton && (
         <SaveButton
-          disabled={!question.canRun()}
+          tooltip={t`You don't have permission to save this question.`}
```

Comment:
Yep, we will have translations for the next maintenance release https://metaboat.slack.com/archives/C864UT5CZ/p1651604160014679

Reply:
I'm late because this has been merged already, but why do we even show the save button? Wasn't it easier to not show it at all and not to bother with adding new tooltip and the new translation string?

cc @alxnddr @kulyk

Reply:
Oh, I see the PR description now.
Ignore me.

Reply:
@nemanjaglumac I tried to address @flamber's concern from [here](https://github.com/metabase/metabase/pull/21823#pullrequestreview-950964705). Since we actually allow editing questions like that it might be totally unclear why users can't save them. Also, it is aligned with [this logic](https://metaboat.slack.com/archives/C0312QQCHLP/p1649353791611779?thread_ts=1649347940.173879&cid=C0312QQCHLP).

cc @mazameli